contributing/ci-docs.txt

```diff
@@ -184,14 +185,13 @@ dev_5_1.
 		This job is used to build the merge auto-generated pages for the
 		dev_5_1 branch of the Bio-Formats documentation
```

Comment:
If you are fixing the autogen branch references to still be develop, it would probably be less confusing if we stick with labelling the BF 5.1 docs as the develop branch for now.